package.json

```diff
@@ -40,40 +40,41 @@
   "devDependencies": {
```

Comment:
I can ditch the `devDependencies` updates from this PR if you'd prefer, but I had done updated things locally while developing, and none of this updates seems to have broken anything.